Makefile

```diff
@@ -0,0 +1,10 @@
+INFO_COLOR=\033[1;34m
+RESET=\033[0m
+BOLD=\033[1m
+TEST ?= $(shell go list ./... | grep -v -e vendor -e keys -e tmp)
+GO ?= GO111MODULE=on $(SYSTEM) go
+
+test: ## Run test
+	@echo "$(INFO_COLOR)==> $(RESET)$(BOLD)Testing$(RESET)"
+	$(GO) test -v $(TEST) -timeout=30s -parallel=4
+	$(GO) test -race $(TEST)
```

Comment:
As a general rule we like to provide/use scripts for testing, etc. in a centralized place (https://github.com/cloudfoundry/cnb-tools) but that said, these scripts would be open to an improvement or even some sort of local override system.